.github/workflows/tests-pa11y.yml

```diff
@@ -35,4 +35,31 @@ jobs:
         run: |
           npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
           "npx http-server dist --port 6006 --silent" \
-          "npx wait-on tcp:6006 && npm run test:pa11y"
\ No newline at end of file
+          "npx wait-on tcp:6006 && npm run test:pa11y" || true
+
+      - name: Archive pa11y report
+        uses: actions/upload-artifact@v3
+        with:
+          name: pa11y-report
+          path: ${{ github.workspace }}/pa11y-ci-report
```

Comment:
We're storing quite a few things now. We should probably define a shorter [retention-period](https://github.com/actions/upload-artifact#retention-period) here.
We can look into this later